share/scripts/inverse/potential_to_gromacs.sh

```diff
@@ -99,7 +100,9 @@ if [[ $tabtype = "non-bonded" ]]; then
       die "${0##*/}: Error table is shorter then what mdp file ($mdp) needs, increase cg.inverse.gromacs.table_end in setting file.\nrlist ($rlist) + tabext ($tabext) > cg.inverse.gromacs.table_end ($tablend)"
     max="$(csg_get_interaction_property max)"
     rvdw="$(get_simulation_setting rvdw)"
-    csg_calc "$max" ">" "$rvdw" && die "${0##*/}: rvdw ($rvdw) is smaller than max ($max)"
+    if [[ $method -ne "iie" ]]; then
```

Comment:
Why is that check not needed for iie?

Reply:
The input (RDF) can be longer than the output (potential).
This is even necessary to make this method work, since there is some issue with the deconvolution in Fourier-space otherwise.
So with iie, a typical setup is to calculate long RDFs and then there is the parameter cg.inverse.iie.cut_off, which defines how long the potential is supposed to be.

Reply:
Ok, maybe a quick comment there.